js/framework/application.js

```diff
@@ -475,12 +475,14 @@ var Application = new Knowledge.Class({
         // from a gresource, so save the file contents to a tmp file.
         [theme_file,] = Gio.File.new_tmp(null);
         theme_file.replace_contents(contents, null, false, 0, null);
-        let [, stdout, stderr, status] = GLib.spawn_command_line_sync(SCSS_COMMAND + theme_file.get_path());
+        let [, stdout_bytes, stderr_bytes, status] = GLib.spawn_command_line_sync(SCSS_COMMAND + theme_file.get_path());
         if (status !== 0) {
-            printerr(new Error(stderr.toString()));
+            let stderr = ByteArray.toString(stderr_bytes)
+            printerr(new Error(stderr));
             System.exit(1);
         }
-        return stdout.toString();
+        let stdout = ByteArray.toString(stdout_bytes);
+        return stdout;
```

Comment:
I would just do `return ByteArray.toString(stdout_bytes);`, same in other places